src/Acl/Client.php

```diff
@@ -175,4 +176,21 @@ public function grantServiceAccountRole($id, $organisation, $role)
         // Send request
         return $this->handleRequest('POST', $requestUrl, $options);
     }
+
+    public function getUserDetails($username)
+    {
+        $filter = '?filters[]=and-user.username-inci-value-' . $username;
```

Comment:
This will work, however `inci` is not the best operand for this use case. https://github.com/rybakdigital/ucc/blob/master/docs/Filter/Criterion.md#operands